src/index.ts

```diff
@@ -174,7 +174,7 @@ export const defaults = (def: MinimatchOptions): typeof minimatch => {
       }
     },
 
-    AST: class AST extends orig.AST {
+    AST: class AST2 extends orig.AST {
```

Comment:
@isaacs You might prefer this. Less confusing.

```suggestion
    AST: class Ast extends orig.AST {
```